### PR TITLE
Manage admin

### PR DESCRIPTION
**Pull request by ThaiNgocHuuBang**

### CODE REVIEW

**Comment by ThaiNgocHuuBang** [comment]
Can't do delete filter with input radio, select all, separate Modal Edit Confirm

File: VMS/Pages/Admin/AccountManagement/AccountAdminSpManagement.razor [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    private void DeselectAll()
+    {
+        IsCheckAll = false;
+    }
+
+    // Modal
+
+    private async Task ShowEditAccountAdminSp()
+    {
+        var options = new ModalOptions()
```
Comment (LongDuongBIT) [inline VMS/Pages/Admin/AccountManagement/AccountAdminSpManagement.razor:145]: No need to init modal options anymore. We already have a method to retrieve it, please see `BlazoredModalOptions.GetModalOptions()`.
Example: https://github.com/UEH-Squad/VMS/blob/a261a3ee729d56186d63a53e3249da8fb7afce6d/VMS/Pages/ActivityInfoPage/Rightrail.razor.cs#L92
Please check other places in this PR.

Comment (31191026415) [reply VMS/Pages/Admin/AccountManagement/AccountAdminSpManagement.razor]: Done

**Comment by LongDuongBIT** [comment]
@hoaitran308 You implement backend for these pages, so please take a look and review this PR.

**Review by hoaitran308 — COMMENTED** [review]
image
Có gì Bằng chỉnh lại cái button radio, cái tâm ở giữa nó bị lệch. Với lại mấy cái dropdown select hình như khi chưa có gì nó sẽ có background trắng, sau khi select thì nó mới xám ấy.

**Comment by hoaitran308** [comment]
@LongDuongBIT @ThaiNgocHuuBang Looks good now!

**Review by LongDuongBIT — APPROVED** [review]

**Pull request merged**